models/insertComment.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../db/connection");
+const { checkArticleExists } = require("../utils/checkArticleExists");
+const { checkUserExists } = require("../utils/checkUserExists");
+
+exports.insertComment = (article_id, content) => {
+  const { username, body } = content;
+
+  if (username === undefined || body === undefined) {
+    return Promise.reject({
+      status: 400,
+      msg: "Missing required fields",
+    });
+  }
+  const checkUser = checkUserExists(username);
```

Comment:
we don't need to check if user exists here - SQL will handle this for us as it's a foreign key constraint!